plugin_tests/common.py

```diff
@@ -29,6 +29,7 @@
 from girder.constants import SortDir
 
 
+JFIFHeader = '\xff\xd8\xff\xe0\x00\x10JFIF'
```

Comment:
Should this be `b\xff\xd8\xff\xe0\x00\x10JFIF`? (Same for the others below?) That would prevent certain undesired behavior between python2 and 3 when doing comparisons.